util/rlp/fuzz/fuzz_targets/append_raw.rs

```diff
@@ -0,0 +1,19 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+#[cfg(feature = "nightly")]
```

Comment:
there is a distinction between `#[attr]` and `#![attr]`. One is applied to the following item, and the other is applied to the enclosing item.